hil-test/.cargo/config.toml

```diff
@@ -0,0 +1,22 @@
+[env]
+DEFMT_LOG="info"
+RUST_LOG = "warn"
+
+[target.'cfg(target_arch = "riscv32")']
+runner    = "probe-rs run"
+rustflags = [
+    "-C", "link-arg=-Tlinkall.x",
+    "-C", "link-arg=-Tembedded-test.x",
+    "-C", "link-arg=-Tdefmt.x", # Comment this line when using log feature
```

Comment:
Do we need the `log` feature at all? If the testing framework is all based on `defmt` it makes sense to me to only use that, I can't imagine a situation where `log` will ever be required in this package?

Reply:
Definetly `log` is not required, just wanted to confirm that it's not useful neither, that's why I put it in the Todo list of the PR. I'm happy to remove it!

Reply:
Note that you can use log or defmt with embedded-test. I use log in my tests, because core::fmt::Debug is implemented for more types than defmt::Format. Also core has more assert macro variants (e.g. assert_matches) than defmt... But up to you to decide...